src/runner.sh

```diff
@@ -99,12 +98,24 @@ function Runner::loadTestFiles() {
   fi
 
   for test_file in "${_FILES[@]}"; do
-    if [[ ! -f $test_file ]];
-    then
+    if [[ ! -f $test_file ]]; then
       continue
     fi
     # shellcheck disable=SC1090
     source "$test_file"
+
+    local duplicates
+    duplicates="$(Helper::getDuplicateFunctions "$test_file")"
+    if [[ -n "$duplicates" ]]; then
+      _DUPLICATED_FOUND=true
+      echo "Duplicate functions found in '$test_file'"
+      IFS=',' read -ra elements <<< "$duplicates"
+      for element in "${elements[@]}"; do
+          echo "- $element"
+      done
+      exit 1
+    fi
```

Comment:
Why do we have a function that returns a string concatenated with the functions and not just whether it is duplicated or not, I have the perception that this logic should be somewhere else and that this is leaking the implementation details into a file that should not have this responsibility

Reply:
I changed this to something like this:
```bash
local duplicates="$(Helper::checkDuplicateFunctions "$test_file")"
if [[ $duplicates == true ]]; then
  State::setDuplicatedFound
  echo "Duplicate functions found in '$test_file'"
  exit 1
fi
```